_employment/job-seekers/start/register.md

```diff
@@ -49,10 +49,6 @@ Contact a [trained specialist](http://www.va.gov/osdbu/verification/assistance/c
         <ul class="small-block-grid-1 medium-block-grid-3 cards small">
 
           <li>
-            <a href="https://www.vip.vetbiz.gov/Public/Logon.aspx"><h5>Registering Your Business</h5></a>
-          </li>
-
-          <li>
             <a href="/employment/job-seekers/start/counseling/"><h5>Entrepreneurship Counseling</h5></a>
           </li>  
```

Comment:
@kantrowitz Your removal of the Registering Your Business piano key looks good, but now there's another issue. When I click on the one remaining piano key, Entrepreneurship Counseling, it takes me to that page, but to the bottom of that page rather than the top with the orange header. Do you know how to fix that? I'll wait to merge until this issue gets resolved. Once you figure it out and update this PR, I'll merge!